include/evmc.h

```diff
@@ -163,6 +161,11 @@ enum evmc_status_code {
     /// An example is RETURNDATACOPY reading past the available buffer.
     EVMC_INVALID_MEMORY_ACCESS = 10,
 
+    /// Exceptions produced by precompiles/system contracts
+    ///
+    /// An example: elliptic curve functions handed invalid EC points
+    EVMC_PRECOMPILE_FAILURE = 11,
```

Comment:
I'm not sure what will return this, because the engines cannot and the clients don't at the moment.

I have a different idea how to solve this in the long term, but this might be an ok short term solution.